### ISSUE TITLE
constant fan setting

### ISSUE THREAD

**Issue by AmL1468** [jxg81/homebridge-actron-que#18]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by AmL1468** [comment]
Hi. Thank you for putting this together. It’s a huge upgrade for the actron que being able to control in HomeKit with automations etc. 

I noticed you don’t use the constant fan mode but said it might be possible to add. Would be great if so. I run this feature overnight as when the rooms cool to set temp, the constant fan keeps air moving and avoids the on/off operation. so it is quiet, less disruptive whilst sleeping and more comfortable. It is much slower than low fan setting so running in fan only wouldn’t replicate. 

If it can be added as a seperate switch, that would avoid having to jump back to the actron app. 

Look forward to seeing this come through at some point.

**Comment by jxg81** [comment]
Yep - i can add this. Ive been using the constant fan setting occasionally myself recently so have had this annoyance

**Comment by AmL1468** [comment]
That’s great. Thank you.

Any idea when it would be ready? No rush, just want to know when to check back for an update.

Thanks again

On 18 Mar 2024, at 2:07 pm, Julian Greensmith ***@***.***> wrote:



Yep - i can add this. Ive been using the constant fan setting occasionally myself recently so have had this annoyance
